content/resources/how-to-guides/how-to-zapier-alert.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to Alert Task to Zapier Webhook
```

Comment:
This title doesn't read well. Also, doc titles should be sentence case.

What about:

- Use InfluxDB to send alerts to Zapier
- Use InfluxDB to trigger Zapier webhooks
- Use InfluxDB tasks to send alerts to Zapier